src/lib.rs

```diff
@@ -21,7 +22,7 @@ fn get_args(current_dir: &str) -> clap::ArgMatches {
             Arg::with_name("input")
                 .help("files or paths")
                 .index(1)
-                .default_value(current_dir)
+                .default_value(current_dir.to_str().unwrap())
```

Comment:
```suggestion
                .default_value_os(current_dir)
```
It will be better this way

Reply:
This is done.